### PR TITLE
Remove ExoPlayer fork dependency

### LINKED ISSUE #27: [ios] Swift is not supported for static libraries
After linking, I get this error when trying to build in XCode:

screen shot 2017-08-15 at 15 48 27

Can I fix this somehow?

### PR DESCRIPTION
**Pull request by mpivchev**
This PR is a follow up of https://github.com/doublesymmetry/KotlinAudio/pull/38

1. This PR removes the ExoPlayer fork dependacy we had, fixing #17 and https://github.com/doublesymmetry/react-native-track-player/issues/1418
2. This PR will also enable bluetooth/physical button media control.
3. It provides Android with proper metadata for the current track, allowing it to populate lock screens and AODs. Can indirectly fix #27, but more tests need to be done.
Example of lock screen: 
image

### CODE REVIEW

**Review by jspizziri — COMMENTED** [review]
Left one comment, but LGTM other than that.

File: android/build.gradle [hunk 1]
```diff
@@ -51,8 +51,8 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.github.DoubleSymmetry:KotlinAudio:v1.0'
-//    implementation "com.github.doublesymmetry:kotlin-audio:1.0"
+//    implementation 'com.github.DoubleSymmetry:KotlinAudio:v1.0'
+    implementation "com.github.doublesymmetry:kotlin-audio:1.1.0"
```
[the remark this answers is not in the document]
Comment (mpivchev) [reply android/build.gradle:55]: KA PR needs to be merged before we can have a remote build.

[the remark this answers is not in the document]
Comment (jspizziri) [reply android/build.gradle]: I guess I'm confused why we're merging in a version with a local dependency? Shouldn't we release KA and use the correct jitpack version here?

**Review by jspizziri — APPROVED** [review]
Lgtm!

**Pull request merged**